pkg/kube/object_patch/validation.go

```diff
@@ -0,0 +1,172 @@
+package object_patch
+
+import (
+	"k8s.io/apimachinery/pkg/util/validation/field"
+)
+
+func validateSpecs(specs []OperationSpec) error {
```

Comment:
We use [openapi schema](https://github.com/flant/shell-operator/blob/master/pkg/hook/config/schemas.go#L11) to validate hooks configurations. Is it better to use the same approach here? WDYT?

Reply:
Oh, I was not aware of that. Yeah, openapi schema makes more sense. Declarative and everything in one place.

Reply:
OpenAPI schema is written. Spec parsing goes through schema validation now.